### PR TITLE
Enhancement: Move BOTPLAY text when middlescroll is on

### PR DESCRIPTION
**Pull request by pahaze**
Is what it says, here's some pics lol

Upscroll:
image

Downscroll:
image

### CODE REVIEW

**Comment by jmxamongusmodder** [comment]
I mean its good and all but i wouldnt use this if you have botplay hidden and act like not botplay. like i do.

**Comment by pahaze** [comment]
> I mean its good and all but i wouldnt use this if you have botplay hidden and act like not botplay. like i do. (aka ratings and score!)

??? BOTPLAY is still visible in middlescroll by default, just right on the arrows?

**Comment by tposejank** [comment]
> > I mean its good and all but i wouldnt use this if you have botplay hidden and act like not botplay. like i do. (aka ratings and score!)
> 
> ??? BOTPLAY is still visible in middlescroll by default, just right on the arrows?

you can make the text not show so easily

**Comment by pahaze** [comment]
> > > I mean its good and all but i wouldnt use this if you have botplay hidden and act like not botplay. like i do. (aka ratings and score!)
> > 
> > 
> > ??? BOTPLAY is still visible in middlescroll by default, just right on the arrows?
> 
> you can make the text not show so easily you just need wiki and lua knowledge

Ah, I see. Didn't know if that was a thing or not in this engine.

**Comment by QuickShark953** [comment]
> Is what it says, here's some pics lol
> 
> Upscroll: image
> 
> Downscroll: image

I definitely agree on moving it. It makes it a lot harder to see the receptors.

**Comment by jmxamongusmodder** [comment]
> > I mean its good and all but i wouldnt use this if you have botplay hidden and act like not botplay. like i do. (aka ratings and score!)
> 
> ??? BOTPLAY is still visible in middlescroll by default, just right on the arrows?

no i hid botplay text and stuff and now it does ratings instead of not doing. like normal gameplay so i can do this without people knowing i had botplay enabled

**Comment by jmxamongusmodder** [comment]
> > > > I mean its good and all but i wouldnt use this if you have botplay hidden and act like not botplay. like i do. (aka ratings and score!)
> > > 
> > > 
> > > ??? BOTPLAY is still visible in middlescroll by default, just right on the arrows?
> > 
> > 
> > you can make the text not show so easily you just need wiki and lua knowledge
> 
> Ah, I see. Didn't know if that was a thing or not in this engine.

no i made it. not in lua.

**Review by jmxamongusmodder — APPROVED** [review]
i approove!

File: source/PlayState.hx [hunk 1]
```diff
@@ -1026,14 +1026,19 @@ class PlayState extends MusicBeatState
 		add(scoreTxt);
 
 		botplayTxt = new FlxText(400, timeBarBG.y + 55, FlxG.width - 800, "BOTPLAY", 32);
+		if(ClientPrefs.downScroll)
+			botplayTxt.y = timeBarBG.y - 78;
+		if(ClientPrefs.middleScroll) {
```
[the remark this answers is not in the document]
Comment (Doritojoey) [reply source/PlayState.hx:1031]: literally the entire point of the pr

**Pull request closed without merging**